__tests__/bin/vip-config-envvar-delete.js

```diff
@@ -13,6 +13,7 @@ import { deleteEnvVarCommand } from '../../src/bin/vip-config-envvar-delete';
 import command from '../../src/lib/cli/command';
 import { deleteEnvVar, validateNameWithMessage } from '../../src/lib/envvar/api';
 import { cancel, confirm, promptForValue } from '../../src/lib/envvar/input';
+// $FlowExpectedError[cannot-resolve-module]
 import { trackEvent } from '../../src/lib/tracker';
```

Comment:
Flow cannot resolve TS modules; we must suppress such errors while converting the codebase.